test/integration_test/stork-test-pod.yaml

```diff
@@ -87,6 +87,8 @@ spec:
       value: aws_access_key_id
     - name: AWS_SECRET_ACCESS_KEY
       value: aws_secret_access_key
+    - name: ENABLE_THREE_CLUSTER_CONFIG
```

Comment:
Would be better to rename this to something like external_test_cluster

Reply:
Renamed.